src/sass/layouts/_header.scss

```diff
@@ -32,6 +40,49 @@
 .icon_next-track {
     background-image: url("../assets/svg/play-next.svg");
 }
+.audio-player-audio-volume {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.volume-container {
+    cursor: pointer;
+    .volume-button {
+        height: 26px;
+        display: flex;
+        align-items: center;
+        .volume {
+            transform: scale(0.7);
+        }
+    }
+    position: relative;
+    z-index: 2;
+    .volume-slider {
+        position: absolute;
+        left: -3px;
+        top: 5px;
+        z-index: -1;
+        width: 0;
+        height: 15px;
+        background: white;
+        box-shadow: 0 0 20px #000a;
+        transition: .25s;
+
+        .volume-percentage {
```

Comment:
do not use nested selectors. just create specific and unique classes. I see that you're trying to use BEM, so just create proper classes for your elements and remove nesting as much as possible